ctapipe/reco/impact.py

```diff
@@ -85,51 +92,65 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
+        default_value="minuit",
+        help="name minimiser to use in the fit",
+    ).tag(config=True)
+
+    atmosphere_profile_name = traits.CaselessStrEnum(
```

Comment:
Just a note for the future: this can be removed and replaced with the new *atmosphere* module once that is merged (see #2000). There, the atmosphere density profile is provided by the EventSource automatically, so no need for it to be repeated here.